content/en/support/_index.md

```diff
@@ -7,6 +7,14 @@ type: docs
 
 Below is a list of companies that offer commercial products and services for Kyverno.
 
+## BlakYaks
+
+[BlakYaks](https://blakyaks.com) are a Microsoft Gold Partner speeding up enterprise container platform adoption, specialising in Azure, DevOps and cloud-native. We focus on building and managing enterprise-grade Azure, container, PaaS, and serverless platforms solutions that are secure, compliant and software-defined.
+
+### Services
+
+BlakYaks are a Nirmata partner that design and deploy Kyverno-managed policies to address the full range of customers Kubernetes policy requirements, whilst leveraging industry best practice throughout.  Engagements start with assessments and end with full policy engine adoption into existing or new cloud-native platforms.
```

Comment:
Since the focus here is on Kyverno, would suggest changing the first line to:  `BlakYaks provides consulting services to design and deploy....`. Thanks!